Resources/public/templates/fields/view/float.hbt

```diff
@@ -0,0 +1,6 @@
+<div class="ez-fieldview-row pure-g">
+    <div class="ez-fieldview-label pure-u">
+        <p class="ez-fieldview-name"><strong>{{ fieldDefinition.names.[eng-GB] }}</strong></p>
+    </div>
```

Comment:
indeed,

```handlebars
<div class="ez-fieldview-value pure-u"><div class="ez-fieldview-value-content">
{{#if isEmpty}}
    This field is empty
{{else}}
    {{ value }}
{{/if}}
</div></div>
```
is much easier to read.
btw @StephaneDiot you should comment on the correct line :p